### PR TITLE
Change absolute paths for relative paths.

### PR DESCRIPTION
**Pull request by domingoruiz**
Hi. To work in local environments it is much better to use relative paths, since for absolute paths to work we have to do several things. With relative paths we can download the code and work directly. I have modified the code and changed all absolute references to relative ones.

UPDATE: In addition, I have added an installation manual in README.md to set up a development environment.
UPDATE: Add Twitter, Reddit, Forum and Github links in config.toml

Thanks you.

A greeting.

### CODE REVIEW

**Comment by hbjydev** [comment]
Would you be able to store the invite link as a variable in the `config.toml` to ensure that we never have to go searching for every instance of `slack.com` again?

An example of how to reference `config.toml` values can be found in the `layouts/partials/meta/site-twittercard.html` file.

**Comment by domingoruiz** [comment]
Ok. I go to do it.

Bye.

File: README.md [hunk 1]
```diff
@@ -21,12 +21,12 @@ mv hugo /usr/bin
 ```
 Finally we install TailwindCSS.
 ```bash
-npm install tailwindcss postcss-cli autoprefixer --save
+npm i -g tailwindcss postcss-cli autoprefixer --save
```
Comment (elysweyr) [inline README.md:24]: Why are you recommending a global installation?

File: public/index.html [hunk 2]
```diff
@@ -87,7 +94,7 @@ <h2 class="text-3xl font-extrabold text-gray-900 dark:text-white sm:text-4xl">
             <dl class="space-y-8 divide-y divide-gray-300 dark:divide-white">
                 <div class="pt-6 md:grid md:grid-cols-12 md:gap-8">
                     <dt class="text-base font-medium text-gray-900 dark:text-white md:col-span-5">
-                        What do you mean, "CentOS has shifted direction?"
+                        What do you mean, "its downstream partner has shifted direction?"
```
Comment (elysweyr) [inline public/index.html:97]: Quite unrelated change.

File: public/index.html [hunk 3]
```diff
@@ -102,7 +109,7 @@ <h2 class="text-3xl font-extrabold text-gray-900 dark:text-white sm:text-4xl">
                     </dt>
                     <dd class="mt-2 md:mt-0 md:col-span-7">
                         <p class="text-base text-gray-500 dark:text-white">
-                            Rocky Linux aims to function as a downstream build as CentOS had done previously, building releases after they have been added to the upstream vendor, not before.
+                            Rocky Linux aims to function as a downstream build as CentOS had done previously, building releases after they have been added by the upstream vendor, not before.
```
Comment (elysweyr) [inline public/index.html:112]: Quite unrelated change.

File: public/index.html [hunk 4]
```diff
@@ -167,10 +174,11 @@ <h2 class="text-3xl font-extrabold text-gray-900 dark:text-white sm:text-4xl">
             &copy; 2020 The Rocky Linux Project.
         </p>
         <p class="mt-2 text-center text-sm text-gray-400">
-            CentOS is a registered trademark of Red Hat, Inc. (“Red Hat”) and has no affiliation with The Rocky Linux Project.
+            CentOS is a registered trademark of Red Hat, Inc. The Rocky Linux Project is not affiliated with, nor endorsed by Red Hat, Inc.
```
Comment (elysweyr) [inline public/index.html:177]: This is also quite unrelated because it's not in the scope of this pull request.

File: public/index.html [hunk 5]
```diff
@@ -1,21 +1,28 @@
...
+<link href="css/main.min.css" rel="stylesheet" />
+<link href="https://fonts.googleapis.com/css?family=Red+Hat+Display:400,400italic,500,500italic,700,700italic,900,900italic" rel="stylesheet" type="text/css" />
+<link href="https://fonts.googleapis.com/css?family=Red+Hat+Text:400,400italic,500,500italic,700,700italic" rel="stylesheet" type="text/css" />
+<link rel="icon" href="img/favicon.ico"/>
+
+<meta name="twitter:title" content="Rocky Linux"/>
+<meta name="twitter:description" content="Rocky Linux is an open enterprise Operating System designed to be 100% bug-for-bug compatible with Enterprise Linux."/>
+<meta name="theme-color" content="#10B981">
+<meta name="twitter:card" content="summary_large_image"/>
+<meta name="twitter:image" content="img/cover.png"/>
```
Comment (elysweyr) [inline public/index.html]: What happened here? You should use the cleanup functionalities of your IDE.

File: README.md [hunk 6]
```diff
@@ -2,4 +2,31 @@
 
 The Rocky Linux landing page, built using [Hugo](https://gohugo.io) and [TailwindCSS](https://tailwindcss.com).
 
-For now, you'll need to run `NODE_ENV=production hugo` for proper PurgeCSS to work in production.
\ No newline at end of file
+#### How to install in centOS 8/ RHEL 8?
+First we need to install nodejs and npm on our machine. For this we will download Node from its website. Once downloaded we unzip and install.
```
Comment (elysweyr) [inline README.md:6]: The official name is "Node.js"
npm ships with Node.JS

File: README.md [hunk 7]
```diff
@@ -2,4 +2,31 @@
...
+
+```bash
+dnf remove nodejs
```
Comment (elysweyr) [inline README.md:9]: https://github.com/nvm-sh/nvm
NVM would make this guide easier and a lot less harmful for a user's environment

**Review by elysweyr — CHANGES_REQUESTED** [review]
Remove unrelated changes and fix the other stuff then it would be fine to me.

File: README.md (hunk 1 above)
Comment (domingoruiz) [reply README.md:24]: Because it's the only way Hugo can use tailwindcss, postcss-cli, and autoprefixer correctly. I read it in Hugo's documentation.

**Review by FoggyMtnDrifter — APPROVED** [review]
Had to fix some conflicts, but PR looks good to go now

**Comment by domingoruiz** [comment]
Ok, very good. Thanks you @ressonix

**Pull request merged**